Portfolio/index.html

```diff
@@ -4,27 +4,33 @@
     <title>Portfolio</title>
     <meta charset="UTF-8" />
     <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <div top="0">
+  <body>
+    <div class="cont1">
+
       <a href="../index.html">Go Back</a>
-    </div>

```

Comment:
Consider breaking down the long paragraph into smaller, more readable chunks for better user experience.